pauthabielf64/pauthabielf64.rst

```diff
@@ -698,7 +700,7 @@ Static Data relocations
   |             |                          |                |                                                     |
   |             |                          |                |                                                     |
   +=============+==========================+================+=====================================================+
-  | 0xE100      | R\_AARCH64\_AUTH\_ABS64  | ``PAUTH(S+A)`` | Signing schema encoded in the contents of the place |
+  | 0x244       | R\_AARCH64\_AUTH\_ABS64  | ``PAUTH(S+A)`` | Signing schema encoded in the contents of the place |
```

Comment:
We are having two relocations named `R_AARCH64_AUTH_ABS64` with different ids (static and dynamic). Should this be named `R_AARCH64_AUTH_ABS64_STATIC` as otherwise there is an ambiguity??

Reply:
This is the same as R_AARCH64_ABS64 which can be both a static and dynamic relocation. I followed that convention as R_AARCH64_AUTH_ABS64 is its PAuth equivalent.

Changing codes is possible but I thought it better to stick the existing convention.

Reply:
It's not the code, it's the name. Essentially we cannot put these two relocations into a single enum – we'd need to rename one entry.

Reply:
Actually I think it is the code, and I've made a mistake, sorry.

The intention is that the name and code remain the same. In the original both had 0xE100, but in the new one I used the first code in the dynamic range, it needed to retain its original 0x244. I'll update the pull request.

Reply:
I've updated the value. I had it right in aaelf64.rst but not here.